WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -90,46 +108,63 @@
-- [ ] Verify able to add audio/video to playlist via share menu
-- [ ] Verify able to add audio/video to playlist via long press context menu
-- [ ] Verify Autoplay is disabled by default
+- [ ] Verify adding audio/video to playlist via share menu
+- [ ] Verify adding audio/video to playlist via long press context menu
+- [ ] Verify Autoplay is enabled by default
 - [ ] Verify with Autoplay enabled, able to play audio/video automatically
 - [ ] Verify with background audio enabled, able to play audio/video with device locked or app in background
+- [ ] Verify Playlist media works with CarPlay on iOS devices
+- [ ] Verify playback gestures are working in Playlist
+- [ ] Verify adding shared folder to playlist using `https://playlist.brave.com/bsa-sample`. Verify all actions on shared folder are working
 
 ## Brave News
 
 - [ ] Verify Brave News opt-in is shown on a clean install
 - [ ] Verify once opted-in Brave News starts showing articles
-- [ ] Verify promoted content is shown on Brave News
 - [ ] Verify inline-content ads are shown on Brave News
-- [ ] Verify able to add custom RSS feed to the list of sources
-- [ ] Verify able to disable content from a particular publisher by long-press on the card
+- [ ] Verify able to add custom RSS feed to the list of sources. Ensure this is working when entering custom URL manually and through the share menu when at a blog page
+- [ ] Verify able to hide content from sources within the `Popular Sources` and `Suggested` categories by long-press on the card
+- [ ] Verify adding Top News Brave widget to home screen will display news articles. Ensure tapping on news articles will load in brave.
+
+## VPN
+
+- [ ] Verify purchasing VPN subscription will install VPN profile and establish connection
+- [ ] Verify tunneling works by navigating to `https://ipinfo.io`
+- [ ] Verify changing server region will update VPN connection
+- [ ] Verify changing protocol from WireGuard to IKEv2
+- [ ] Verify `Reset Configuration` will reset everything but the region selected
 
 ## Bravery settings
 
-- [ ] Check that HTTPS Everywhere works by loading `http://https-everywhere.badssl.com/`
-- [ ] Turning HTTPS Everywhere off or shields off both disable the redirect to `https://https-everywhere.badssl.com/`
 - [ ] Check that block ad and unblock ad works on `http://slashdot.org`
-- [ ] Check that toggling to blocking and allow ads works as expected
+- [ ] Check that toggling to blocking and allow ads works as expected. Ensure pre-roll ads are removed on `youtube.com`
 - [ ] Verify that tapping through a cert error in `https://badssl.com` works
-- [ ] Verify that Safe Browsing works `https://www.raisegame.com`
-- [ ] Enable `Block Scripts` globally from `Settings`.  Visit `https://twitter.com`, nothing should load. Tap on `Shields` and disable `Block Scripts`; the page should load properly.
+- [ ] Verify Safe Browsing protection on `https://testsafebrowsing.appspot.com/` in the iOS
```

Comment:
Please confirm if these tests are applicable for iOS. These are specific for Chromium checks

Reply:
Yes, these tests will work for testing blocking dangerous sites on iOS. There is a specific section for iOS on the page. It was suggested by the privacy team for iOS testing.